### PR TITLE
Add more context for admin stacks

### PR DESCRIPTION
**Pull request by jubranNassar**
# Description of the change

I added an info box in the admin section of the stack settings page to provide more clarity for admin stacks.

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [ ] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [ ] The preview looks fine.
- [ ] The tests pass.
- [ ] The commit history is clean and meaningful.
- [ ] The pull request is opened against the `main` branch.
- [ ] The pull request is no longer marked as a draft.
- [ ] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/concepts/stack/stack-settings.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ Another pattern we've seen is stacks exporting their outputs as a [context](../c
 
 If this sounds interesting and you want to give it a try, please refer to the [help article exclusively dedicated to Spacelift's Terraform provider](../../vendors/terraform/terraform-provider.md).
 
+!!! info
```
Comment (jmfontaine) [inline docs/concepts/stack/stack-settings.md:21]: [The first sentence](https://github.com/spacelift-io/user-documentation/pull/136/files#diff-a10a10f63b2797918f4e6756fcf86764fb175ad6c6d8d2ef94513cc2cbcf3986L13) basically says that.

Should you not update that sentence instead?

> This setting indicates whether a stack has administrative privileges for the [Space](../spaces/README.md) it is in.

Comment (jubranNassar) [reply docs/concepts/stack/stack-settings.md:21]: update the first sentence instead of the info box?

File: docs/concepts/spaces/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ In Spacelift, this can be achieved by splitting your account into multiple Space
 
 Spaces are sets that can be filled with various kinds of Spacelift entities: [Stacks](../stack/README.md), [Policies](../policy/README.md), [Contexts](../configuration/context.md), [Modules](../../vendors/terraform/module-registry.md), [Worker Pools](../worker-pools.md), and [Cloud Integrations](../../integrations/cloud-providers/README.md).
 
-Initially, you start with a `root` and a `legacy` space. The `root` space is the top-level space of your account, while the `legacy` space exists for backward compatibility with pre-spaces RBAC. You can then create more spaces, ending up with a big tree of segregated environments.
+Initially, you start with a `root` and a `legacy` space. The `root` space is the top-level space of your account, while the `legacy` space exists for backward compatibility with pre-spaces RBAC. [You can then create more spaces](https://spacelift-user-documentation-pr-136.onrender.com/concepts/spaces/creating-a-space.html), ending up with a big tree of segregated environments.
```
Comment (jmfontaine) [inline docs/concepts/spaces/README.md:13]: Please link to the documentation, not the preview.

File: docs/concepts/stack/stack-settings.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ This article covers all settings that are set **directly on the stack**. It's im
 
 ### Administrative
 
-This setting indicates whether a stack has administrative privileges. Runs executed by administrative stacks receive an API token that gives them administrative access to a subset of the Spacelift API used by our[Terraform provider](../../vendors/terraform/terraform-provider.md), which means they can create, update and destroy Spacelift resources.
+This setting indicates whether a stack has administrative privileges within the [space](https://spacelift-user-documentation-pr-136.onrender.com/concepts/spaces/index.html) it lives in. Runs executed by administrative stacks receive an API token that gives them administrative access to a subset of the Spacelift API used by our [Terraform provider](../../vendors/terraform/terraform-provider.md), which means they can create, update and destroy Spacelift resources.
```
Comment (jmfontaine) [inline docs/concepts/stack/stack-settings.md:13]: Please link to the documentation, not the preview.

**Review by jmfontaine — CHANGES_REQUESTED** [review]

File: docs/concepts/spaces/README.md (hunk 2 above)
Comment (jubranNassar) [reply docs/concepts/spaces/README.md]: done

**Review by jmfontaine — CHANGES_REQUESTED** [review]

File: docs/concepts/spaces/README.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ In Spacelift, this can be achieved by splitting your account into multiple Space
 
 Spaces are sets that can be filled with various kinds of Spacelift entities: [Stacks](../stack/README.md), [Policies](../policy/README.md), [Contexts](../configuration/context.md), [Modules](../../vendors/terraform/module-registry.md), [Worker Pools](../worker-pools.md), and [Cloud Integrations](../../integrations/cloud-providers/README.md).
 
-Initially, you start with a `root` and a `legacy` space. The `root` space is the top-level space of your account, while the `legacy` space exists for backward compatibility with pre-spaces RBAC. You can then create more spaces, ending up with a big tree of segregated environments.
+Initially, you start with a `root` and a `legacy` space. The `root` space is the top-level space of your account, while the `legacy` space exists for backward compatibility with pre-spaces RBAC. [You can then create more spaces](https://docs.spacelift.io/concepts/spaces/creating-a-space), ending up with a big tree of segregated environments.
```
[the remark this answers is not in the document]
Comment (jubranNassar) [reply docs/concepts/spaces/README.md:13]: ahh yes, I just fixed it with next commit

**Review by jmfontaine — APPROVED** [review]

**Pull request merged**